File: ejercicios parte 1/1)programas simples/8parte decimal/parte_decimal.cpp
#include <iostream>
using namespace std;
int main(){
    float numero;
    int numeroInt;

    cout<<"Ingrese un número: ";
    cin>>numero;
    
    numeroInt = numero;
    numero = numero - numeroInt;

    cout<<"Decimal: "<<numero<<endl;
    return 0;
}

File: ejercicios parte 1/1)programas simples/8parte decimal/parte_decimal.java
import java.util.Scanner;

public class parte_decimal {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int numeroInt;
            float numero;
            System.out.print("Ingrese un número: ");
            numero = scan.nextFloat();
            numeroInt = ((int)numero);
            numero = numero -numeroInt;
            System.out.println("La parte decimal de este número fue: " + numero);
        }
    }    
}
